frameworks/javascript/cw-2.js

```diff
@@ -45,59 +44,52 @@ try {
     }
   };
 
-  var _expect = function(passed, failMsg, options) {
-    options = options || {};
-
+  const _expect = function(passed, failMsg = "Value is not what was expected", options = {}) {
     if (passed) {
-      var successMsg = "Test Passed";
-      if (options.successMsg) {
+      let successMsg = "Test Passed";
+      if (options.hasOwnProperty("successMsg") && typeof options.successMsg === "string") {
         successMsg += ": " + options.successMsg;
       }
       Test.display.write("PASSED", successMsg);
 
-      if (options.passed) {
+      if (options.hasOwnProperty("passed") && options.passed) {
         options.passed(successMsg, options);
       }
-    }
-    else {
-      failMsg = failMsg || 'Value is not what was expected';
-
+    } else {
```

Comment:
> Remove "hanging elses"

```javascript
}
else {
```

This style is preferred by @jhoffner and we have a ESLint rule so please don't change this.